packages/mdc-theme/README.md

```diff
@@ -46,13 +46,13 @@ npm install --save @material/theme
 
 ## Usage
 
-### Sass
+### Themeing
 
-#### Changing the theme colors
+MDC Theme mixins use CSS custom properties for browsers that support it, with a fallback to a pre-processed 
+static color if they don't. This enables runtime theming if CSS properties are available.
 
-MDC Theme makes it quite easy to change the theme colors for your application, and have the changes apply to all MDC-Web
-components. Simply define the three theme color variables before importing `mdc-theme` or any MDC-Web components that rely
-on it:
+To set the theme colors for your application, simply define the three theme color variables before importing 
```

Comment:
Define or override? I feel the latter is more appropriate as we do provide default theme color. How about:

> MDC Theme makes it easy to develop your brand colors. You can simply override the default theme color by through Sass variable or CSS custom properties. In the latter case, it enables runtime theming.
> 
> **A note for Sass variables**, you need you define the three theme color variables before importing `mdc-theme` or any MDC-Web components that rely on it, like following: